web/src/portico/header.ts

```diff
@@ -64,7 +66,9 @@ $(() => {
             e.preventDefault();
             e.stopPropagation();
             const labelID = $(e.currentTarget).attr("for");
-            $(`#${CSS.escape(labelID)}`).trigger("click");
+            if (labelID) {
+                $(`#${CSS.escape(labelID)}`).trigger("click");
+            }
```

Comment:
I think this might be better written as:
```
if (!labelID) {
    throw new Error("...")
}
$(`#${CSS.escape(labelID)}`).trigger("click");
```

Generally we don't want to suppress failures silently -- that makes the developer experience worse, not better.

Reply:
@timabbott The question is what's gonna handle this error?

Reply:
No, errors are reported to the folks who run the Zulip server via Sentry for investigation. More importantly, given the nature of this code path, the most likely situation where this is going to come up is when someone is doing some refactoring on this page and testing it, and in that case the errors appear in a popup. 

With your approach, it'll just silently not work.